raiden/tests/integration/fixtures/smartcontracts.py

```diff
@@ -48,7 +55,18 @@ def deploy_all_tokens_register_and_return_their_addresses(
     if register_tokens:
         for token in token_addresses:
             registry = deploy_service.token_network_registry(token_network_registry_address)
-            registry.add_token(token_address=token, given_block_identifier='latest')
+            if contract_manager.contracts_version == DEVELOPMENT_CONTRACT_VERSION:
+                registry.add_token_with_limits(
+                    token_address=token,
+                    channel_participant_deposit_limit=RED_EYES_PER_CHANNEL_PARTICIPANT_LIMIT,
+                    token_network_deposit_limit=RED_EYES_PER_TOKEN_NETWORK_LIMIT,
+                    given_block_identifier='latest',
+                )
+            else:
+                registry.add_token_without_limits(
+                    token_address=token,
+                    given_block_identifier='latest',
```

Comment:
In this file you seem to still have 'latest' blocks. Will you use blockhash as we discussed?

Reply:
I plan to create a separate issue for converting tests to use the actual blockhash instead of `latest`... but that's going to be in a different PR/issue.

Reply:
Fair enough! Please do though, I normally would ask for the issue to be created first so we don't forget but let's get this merged since it's open for a while.

Reply:
Created: https://github.com/raiden-network/raiden/issues/3619